src/components/EditCoursePage/index.jsx

```diff
@@ -382,6 +383,11 @@ class EditCoursePage extends React.Component {
         />
 
         <PageContainer>
+          { showForm && !editable && <StatusAlert
+            alertType="secondary"
+            message="You have permission to view this course, but not edit. If you would like to edit the course, please contact your project coordinator."
```

Comment:
This might only be a temporary message, but we should make sure we remember that eventually they should be able to just contact a current editor on the course to be added as one.

Reply:
Yes. I figured when we add the editor UI, we can fix this up.